server/auth/auth_handler_factory.ts

```diff
@@ -80,7 +81,7 @@ export async function getAuthenticationHandler(
         throw new Error(`Unsupported authentication type: ${currType}`);
     }
   } else {
-    if (config.auth.multiple_auth_enabled) {
+    if (authType.length > 1) {
```

Comment:
do we need `multiple_auth_enabled` anymore?

Reply:
I don't think we need it anymore in this statement, because if it has more than 1 option available it should have multiple authentication enable.